### DISCUSSION TITLE
Set up a foundation or be part of a foundation

### DISCUSSION THREAD

**Discussion by sirinath** [GraphiteEditor/Graphite#1464] [Ideas/Feedback]
Set up a foundation to collect donation and perhaps hire people if you get reasonably large donations.

**Comment by Keavon** [comment]
You may donate directly to the Graphite organization [here](https://github.com/sponsors/GraphiteEditor).

**Reply by sirinath** [reply]
Having a foundation or being part of it is more about sustainability. Also it will be easy to attach cooperate sponsors and perhaps code submissions. Also there is some continuity of the project current founders in case of unforeseen circumstances.
